### ISSUE TITLE
No Windows 64bit nightlies for 26 Feb

### ISSUE THREAD

**Issue by chrisjlocke** [sqlitebrowser/sqlitebrowser#1770]
Details for the issue
--------------------
The 32bit Windows nightlies are available, but not the 64bit versions.  No .ZIP or .MSI.

#### What did you do?
Went to download the nightlies

#### What did you expect to see?
The nightlies.

#### What did you see instead?
No nightlies.

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [ ] 3.11.1
- [ ] 3.11.0
- [ ] 3.10.1
- [ ] Other: ___

#### Did you also

- [ ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ ] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by chrisjlocke** [comment]
![image](https://user-images.githubusercontent.com/7872218/53408240-f42d8b00-39b5-11e9-9779-295f00de2d6b.png)

**Comment by justinclift** [comment]
Thanks @chrisjlocke, will take a look. :smile:

**Comment by justinclift** [comment]
Weirdly, the Task Scheduler is saying the last run was terminated by the user.  Haven't logged into that box today, so it's definitely not meaning something I manually did.

Guessing it means the option I set for (paraphrasing here) "kill the process if it takes more than 4 hours" kicked in.  I've just manually re-launched the win64 build process, so lets see how that goes.  If it seems to go wrong, I'll investigate further. :smile:
